server/types.graphql

```diff
@@ -1,23 +1,86 @@
-# The following types define the data model of the example service
-# based on which the GraphQL API is generated
-
 type User @model {
   id: ID! @isUnique
-  name: String
-  dateOfBirth: DateTime
+  email: String! @isUnique
+  password: String! @isUnique
+  dateOfBirth: DateTime!
+  name: String!
+  points: Int! @default(value: 0)
+  profilePicture: File @relation(name: "UserPicture")
+  cardBarcode: String
+  favouriteBeers: [Beer!]! @relation(name: "UserBeers")
+  pointsHistory: [Point!]! @relation(name: "UserPoint")
+  favouriteEvents: [Event!]! @relation(name: "UserEvents")
+  redeemHistory: [Redeem!]! @relation(name: "UserRedeems")
+}
+
+type File @model {
+  id: ID! @isUnique
+  name: String!
+  url: String!
+  contentType: String!
+  user: User! @relation(name: "UserPicture")
+}
+
+type Point @model {
+  id: ID! @isUnique
+  user: User! @relation(name: "UserPoint")
+  beer: Beer! @relation(name: "BeerPoint")
```

Comment:
BeerPoints? 
UserPoints?

these are one-many relationships, be careful with your naming convention.

Reply:
Fixed